libpromises/evalfunction.c

```diff
@@ -10303,6 +10366,13 @@ static const FnCallArg DATATYPE_ARGS[] =
     {CF_BOOL, CF_DATA_TYPE_OPTION, "Enable detailed type decription"},
     {NULL, CF_DATA_TYPE_NONE, NULL}
 };
+static const FnCallArg IS_DATATYPE_ARGS[] =
+{
+    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "Variable identifier"},
+    {CF_ANYSTRING, CF_DATA_TYPE_STRING, "Type"},
+    {CF_BOOL, CF_DATA_TYPE_OPTION, "Enable detailed type decription"},
```

Comment:
`detail` should not be needed for `is_type()`, it can be deduced by looking at the type string (if it's 1 or 2 words).